cluster-operator/src/test/java/io/strimzi/operator/cluster/operator/assembly/KafkaAssemblyOperatorTest.java

```diff
@@ -693,10 +685,6 @@ public ReconciliationState createReconciliationState(Reconciliation r, Kafka ka)
         ops.createOrUpdate(new Reconciliation("test-trigger", Kafka.RESOURCE_KIND, kafkaNamespace, kafkaName), kafka)
             .onComplete(context.succeeding(v -> context.verify(() -> {
 
-                // No metrics config  => no CMs created
-                Set<String> logsAndMetricsNames = new HashSet<>();
-                logsAndMetricsNames.add(KafkaCluster.metricAndLogConfigsName(kafkaName));
-
```

Comment:
After @scholzj's comments about this code removal a couple of PR codes ago, I dug into Git, and the test code associated with this left over setup was commented out in 2018 and then removed in 2019 (IIRC).

Reply:
@scholzj  Are you okay with this deletion, and the reasoning behind it?